runrestic/restic/tools.py

```diff
@@ -29,9 +29,7 @@ def __init__(
 
     def run(self) -> List[Dict[str, Any]]:
         for command in self.commands:
-            if isinstance(command, str):
-                command = [command]
-            logger.debug(f'Spawning "{command}"')
+            logger.debug("Spawning %s", command)
```

Comment:
why did you convert the f-string?

Reply:
I changed it because pylint rasies a warning  that f-strings should be avoided in log message, see https://pylint.pycqa.org/en/latest/user_guide/messages/warning/logging-fstring-interpolation.html

I think in this specific case f-strings are not really ans issue, but I just tend to follow this adviss. If you don't like it, I can revert back.

Reply:
Found back an example where a %s style log works but f-string crashes:
https://github.com/PyCQA/pylint/issues/1788

So to not decide case by case if f-string is save or not, i's easier to avoid it.
Other reason against f-string is some potential optimization in case the variable need to be formatted, but this is less important in my view.

Reply:
cool, didn't know - thanks for the explanation!